core/src/main/resources/hudson/model/Computer/builds.properties

```diff
@@ -20,4 +20,5 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 
-title=Build History on {0}
\ No newline at end of file
+title=Build History on {0}
+disclaimer=This does not include stages from pipeline jobs.
```

Comment:
does core depend on `pipeline` to hardcode additional plugin job type naming?

Reply:
It isn't ignorant of it, including some whitelisting:
https://github.com/jenkinsci/jenkins/search?q=pipeline&unscoped_q=pipeline

If you really think that pipeline should hook these files and can provide an example of how to do it, I suppose I could look into it.

Reply:
We have huge instance with custom job type and we are not using any pipeline plugins, like there are a lot of people still using freestyle jobs. Core makes confusion for users.

Reply:
@KostyaSha, would it make more sense for you if that would be a link pointing to https://jenkins.io/solutions/pipeline/ ?

Reply:
```suggestion
disclaimer=This history does not guarantee to include all subtasks being executed on the node, e.g. Jenkins Pipeline subtasks will not be displayed.
```

Reply:
`does not guarantee to` -> `is not guaranteed to`

The former is wrong for some reason I can't quite explain.